contracts/strategies/allocation/AllocationWithVotingBase.sol

```diff
@@ -116,44 +116,53 @@ contract AllocationWithVotingBase is IAllocationStrategy, Ownable, Initializable
     }
 
     modifier onlyPoolAdmin() {
-        if (!registry.isOwnerOrMemberOfIdentity(allo.pools[_poolId].identityId, msg.sender)) {
+        (bytes32 identityId,,,) = allo.pools(poolId);
+        if (!registry.isOwnerOrMemberOfIdentity(identityId, msg.sender)) {
             revert UNAUTHORIZED();
         }
         _;
     }
 
     /// @notice Constructor
-    function AllocationWithVotingBase(bytes32 _identityId_, address _allo) {
+    function allocationWithVotingBaseConstructor(bytes32 _identityId_, address _allo) public {
```

Comment:
we could likely rename as this constructor.

Reply:
How do you mean?